src/main/java/com/exasol/clusterlogs/LogEntryPatternVerifier.java

```diff
@@ -6,6 +6,21 @@
  */
 interface LogEntryPatternVerifier {
 
+    /**
+     * A {@link LogEntryPatternVerifier} which simulates to find any log message.
```

Comment:
```suggestion
     * A {@link LogEntryPatternVerifier} which accepts any log message.
```
❓ 
Why do we need this in production code?